featureflags/polling.py

```diff
@@ -28,6 +28,11 @@ def __init__(self, client: AuthenticatedClient, config: Config,
 
     def run(self):
         if not self.__running:
+            if self.__config.pull_interval < 60:
+                log.info("Pull Interval must be greater than or equal "
```

Comment:
nitpick here, but can you update this to a `log.warn`. If the user has set minimum log level to exclude info, we'd still want them to know an incorrect param was set.